airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/github.py

```diff
@@ -102,27 +102,36 @@ def github_metadata_definitions(context):
     return Output(metadata_definitions, metadata={"preview": [md.json() for md in metadata_definitions]})
 
 
+def entry_should_be_on_gcs(metadata_entry: LatestMetadataEntry) -> bool:
```

Comment:
readability helper. 
We might eventually want to change the stale metadata detection logic to get alerts when a RC does not end up in the registry

Reply:
Yes. I'll file a different issue to update the stale metadata detection to consider release candidates.